frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json

```diff
@@ -343,7 +343,7 @@
   "Storage Cluster exists": "Storage Cluster exists",
   "Back to operator page": "Back to operator page",
   "Go to cluster page": "Go to cluster page",
-  "clusterExistText": "A storage cluster <1>{{clusterName}}</1> is already created.<3></3>You cannot create another storage cluster.",
+  "clusterExistText": "A storage cluster <1>{{clusterName}}</1> already exists.<3></3>You cannot create another storage cluster.",
```

Comment:
Why this change ?

Reply:
I think this might have cropped up when I ran `yarn run i18n` on this branch a while ago. Although I didn't make this change, I thought this was something the aforementioned command did, not sure why.

Should I go ahead and revert it?

Reply:
yarn run i18n would not add this. Possibly some rebase issue.
Please rebase your PR with latest master, it will vanish.